site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-type.vm

```diff
@@ -193,6 +193,11 @@
                             #set($siteType = false)
                             #set($siteType = $!templateSite.type)
 
+                            #if (
+                                ($siteType == $projectSiteType && $canAddProject) || 
+                                ($courseSiteTypeStrings.contains($siteType) && $canAddCourse) ||
+                                ($siteType == $portfolioSiteType && $canAddPortfolio)
+                             )
                             #if($siteTypePrev != $siteType)
```

Comment:
Can this second if statement be combined with your new if statement so we don't have to worry about changing indentation?